.NET/Microsoft.Recognizers.Text.DateTime/Swedish/Parsers/SwedishDateParserConfiguration.cs

```diff
@@ -136,6 +136,12 @@ public SwedishDateParserConfiguration(ICommonDateTimeParserConfiguration config)
 
         public Regex BeforeAfterRegex { get; }
 
+        public Regex TasksModeDurationToDatePatterns { get; }
+
+        public Regex TasksModeNextMonthRegex { get; }
```

Comment:
These two "next regexes" are not needed, right?

Reply:
right, removed them in next commit